numpy/core/arrayprint.py

```diff
@@ -3,9 +3,10 @@
 $Id: arrayprint.py,v 1.9 2005/09/13 13:58:44 teoliphant Exp $
 
 """
-__all__ = ["array2string", "array_str", "array_repr", "set_string_function",
-           "set_printoptions", "get_printoptions", "printoptions",
-           "format_float_positional", "format_float_scientific"]
+__all__ = ["array2string", "array_str", "array_repr", "array_format",
+           "set_string_function", "set_printoptions", "get_printoptions",
+           "printoptions", "format_float_positional",
+           "format_float_scientific"]
```

Comment:
Why is it necessary to expose `array_format` here? You can still import it directly without exposing it in `__all__`.